mlir/lib/Conversion/TosaToRock/TosaToRock.cpp

```diff
@@ -111,7 +111,7 @@ static Value expandTensor(ConversionPatternRewriter &rw, Operation *op,
 static std::tuple<StringAttr, std::optional<uint32_t>, rock::GemmFeatures>
 getArchAttributes(Operation *op, Type inputType) {
   auto func = op->getParentOfType<func::FuncOp>();
-  auto mod = func->getParentOfType<ModuleOp>();
+  //auto mod = func->getParentOfType<ModuleOp>();
```

Comment:
... this shouldn't be dead code, I think we can still have cases where the module is `arch`'d and the function isn't?

Reply:
I didn't write it, I just got tired of the unused-variable warning.